tests/test_user_manager.py

```diff
@@ -5,18 +5,18 @@
 
 from faker import Faker
 from pyrogram.types import User
+from pytest import fixture
 
 from src.korone.database import Database
-from src.korone.database.manager import UserManager, Clause, Column
-
-from pytest import fixture
+from src.korone.database.manager import Clause, Column, UserManager
```

Comment:
Importing from src is not needed. For testing, one should first build the package and install it in its local environment, and then one should be able to run the tests without having to do this hack.